### PR TITLE
Update Hotwire tutorial description

### PR DESCRIPTION
**Pull request by scuml**
Adds picture
Updates title and description for Hotwire talk

### CODE REVIEW

**Review by tim-schilling — APPROVED** [review]
I think this is great! I do think the overview of the topics you plan on covering is a great as well. Let me know if you'd rather proceed with or without that list.

**Comment by scuml** [comment]
@tim-schilling Could I get your thoughts on the updated `Tutorial Schedule` section?

File: _schedule/tutorials/2023-10-08-13-00-t2-hotwire-a-refreshing-approach-to-the-front-end-that-keeps-django-the-star.md [hunk 1]
```diff
@@ -21,21 +21,54 @@ slug: hotwire-a-refreshing-approach-to-the-front-end-that-keeps-django-the-star
...
+This may look like too much to do in such a short time, but Hotwire makes things so simple, you'll be amazed what you can accomplish.  We will likely finish early.  There will be a 5-10 minute break between each segment.
+
+#### Hour 1
+
+* Introduction:  Why are things so complex and how did we get here?
+* Dive right in - Create a single page app in Django
+* Overview of Hotwire Turbo - What is it?
+* How to use Django forms with Turbo
+* Make lazy-loading tabs
+* Keep that dopamine coming with an infinite scrolling page
```
Comment (tim-schilling) [inline _schedule/tutorials/2023-10-08-13-00-t2-hotwire-a-refreshing-approach-to-the-front-end-that-keeps-django-the-star.md:60]: :rocket:

**Review by tim-schilling — APPROVED** [review]
Looks great to me!

**Comment by tim-schilling** [comment]
@scuml this should be live in 5-10 minutes

**Comment by scuml** [comment]
Looks good on the site.  Thanks for reviewing.

**Pull request merged**